lib/fs.js

```diff
@@ -1842,7 +1842,7 @@ ReadStream.prototype.close = function(cb) {
       this.once('open', close);
       return;
     }
-    return process.nextTick(this.emit.bind(this, 'close'));
+    return process.nextTick((fd) => this.emit('close', fd));
```

Comment:
I think this should just be `() => this.emit('close')`?

@trevnorris, could you confirm?